eZ/Publish/Core/Repository/Permission/PermissionsCriterionResolver.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\Repository\Permission;
+
+use eZ\Publish\API\Repository\PermissionCriterionResolver as APIPermissionCriterionResolver;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion\LogicalAnd;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion\LogicalOr;
+use eZ\Publish\API\Repository\Values\User\Limitation;
+use eZ\Publish\API\Repository\PermissionResolver as PermissionResolverInterface;
+use eZ\Publish\Core\Repository\Helper\LimitationService;
+use RuntimeException;
+
+/**
+ * Implementation of Permissions Criterion Resolver.
+ *
+ * @since 6.12
```

Comment:
PR is targeted to 6.7, so I'm not sure about this

Reply:
Yes, I'll need to adapt this.

Reply:
Yes, I'll need to adapt this.